postgresql_test.go

```diff
@@ -22,3 +24,47 @@ func init() {
 		},
 	)
 }
+
+type PostgreSQLTableNameTestCase struct {
+	FileName string
+	Escaped  string
+}
+
+func testEscapeTableName(t *testing.T) {
```

Comment:
I think this is not running because tests should start with capital letter (`TestEscapeTableName`).

Let's move this test to `testfixtures_test.go` file. For now it's done only for PG, but I'll do for other databases, too:

```go
func TestQuoteKeyword(t *testing.T) {
	tests := []struct{
		Helper    Helper
		Keyword   string
		Expected  string
	}{
		{&PostgreSQL{}, `posts_tags`, `"posts_tags"`},
		{&PostgreSQL{}, `test_schema.posts_tags`, `"test_schema"."posts_tags"`},
	}

	for _, test := range tests {
		assert.Equal(t, test.Expected, test.Helper.quoteKeyword(test.Keyword))
	}
}
```